tests/utils/test_download_utils.py

```diff
@@ -0,0 +1,97 @@
+import pathlib
+
+import pytest
+
+from folktables import exceptions
+from folktables.utils import download_utils
+from folktables.utils import files_resources
+
+MOCK_URL = 'http://test.com'
+
+
+class DownloadTestResource(files_resources.FilesResource):
```

Comment:
Can we directly instantiate an object of type `FilesResource` here?

Reply:
We'll be able to after I revert from having `FilesResource` being an `ABC`.